apps/src/blockly/constants.ts

```diff
@@ -65,6 +66,7 @@ export const Renderers = {
   DEFAULT: 'cdo_renderer_thrasos',
 };
 
+export const NAVIGATION_CURSORS = ['default', 'basic', 'line'];
```

Comment:
Why not refer to these as cursor types? (Since they're not cursor instances.)

Reply:
Sure!